src/app.js

```diff
@@ -1 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
+
+const inquirer = require('inquirer');
+const fs = require('fs/promises');
+
+const commandHandler = async() => {
+  const command = await inquirer
+    .prompt([{
```

Comment:
process.argv gives you access to params. Inquirer seems too complicated in this case for me))